pytorch_lightning/strategies/hpu_parallel.py

```diff
@@ -77,7 +90,7 @@ def setup_environment(self) -> None:
     def determine_ddp_device_ids(self) -> None:
         return None
 
-    def pre_configure_ddp(self):  # type: ignore
+    def _pre_configure_ddp(self):  # type: ignore
```

Comment:
Why prefix it with `_`? Now it doesn't match the name in `ddp.py`

Reply:
I would like to keep it with the child and use it for only torch where static_graph is not a ddp param